tests/interpreters/test_onnx_interpreter.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+
+from xdsl.dialects import onnx
+from xdsl.dialects.builtin import ModuleOp, TensorType, f32
+from xdsl.interpreter import Interpreter
+from xdsl.interpreters.shaped_array import ShapedArray
+from xdsl.utils.test_value import TestSSAValue
+
+found_numpy = pytest.importorskip(
+    "numpy", reason="numpy is an optional dependency in xDSL"
+)
+import numpy as np  # noqa: I001, E402
+from xdsl.interpreters.onnx import OnnxFunctions  # noqa: I001, E402
+
+interpreter = Interpreter(ModuleOp([]))
+interpreter.register_implementations(OnnxFunctions())
+
+
+def test_onnx_add():
+    op = onnx.Add(
+        TestSSAValue(TensorType(f32, [2, 3])),
+        TestSSAValue(TensorType(f32, [2, 3])),
+        res_type=TensorType(f32, [2, 3]),
+    )
+
+    a = ShapedArray(np.array([1, 2, 3, 4, 5, 6]), [2, 3])
+    b = ShapedArray(np.array([1, 4, 2, 5, 3, 6]), [2, 3])
+    c = ShapedArray(np.array([0, 0, 0, 0, 0, 0]), [2, 3])
+
+    c = interpreter.run_op(op, (a, b, c))
```

Comment:
Looks like a bug in the interpreter, we shouldn't allow passing more arguments to this than the number of operands in the op, I'll fix this later, in the meantime we should remove the c from the argument
```suggestion
    c = interpreter.run_op(op, (a, b))
```